src/test/java/jp/ikedam/jenkins/plugins/extensible_choice_parameter/IntegrationTest.java

```diff
@@ -1,55 +0,0 @@
-/*
- * The MIT License
- *
- * Copyright (c) 2018 IKEDA Yasuyuki
- *
- * Permission is hereby granted, free of charge, to any person obtaining a copy
- * of this software and associated documentation files (the "Software"), to deal
- * in the Software without restriction, including without limitation the rights
- * to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
- * copies of the Software, and to permit persons to whom the Software is
- * furnished to do so, subject to the following conditions:
- *
- * The above copyright notice and this permission notice shall be included in
- * all copies or substantial portions of the Software.
- *
- * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
- * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
- * FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
- * AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
- * LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
- * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
- * THE SOFTWARE.
- */
-
-package jp.ikedam.jenkins.plugins.extensible_choice_parameter;
-
-import static org.junit.Assert.assertEquals;
-
-import org.junit.Rule;
-import org.junit.Test;
-import org.jvnet.hudson.test.Issue;
-import org.jvnet.hudson.test.recipes.LocalData;
-
-import hudson.model.FreeStyleBuild;
-import hudson.model.FreeStyleProject;
-import hudson.model.TaskListener;
-
-/**
- * Tests using several {@link ChoiceListProvider}s
- */
-public class IntegrationTest {
-    @Rule
-    public ExtensibleChoiceParameterJenkinsRule j = new ExtensibleChoiceParameterJenkinsRule();
-
-    @Test
-    @Issue("JENKINS-49210")
-    @LocalData
-    // This test explains why we need hudson.remoting.ClassFilter file with Jenkins 2.102+
-    public void testJEP200() throws Exception {
-        FreeStyleProject p = j.jenkins.getItemByFullName("JobWithSublist", FreeStyleProject.class);
-        FreeStyleBuild b = j.assertBuildStatusSuccess(p.scheduleBuild2(0));
-        assertEquals("ChoiceWithSublist1", b.getEnvironment(TaskListener.NULL).get("ChoiceWithSublist"));
-        assertEquals("GlobalChoiceWithSublist1", b.getEnvironment(TaskListener.NULL).get("GlobalChoiceWithSublist"));
-    }
```

Comment:
The sublist class has moved in Java 11. Since recent Jenkins core versions require Java 11, this implies we can no longer support upgrade from 1.4.2 which saved data with the Java 8 class name. As such, bumping the compatible since version to 1.5.0 and removing any migration logic.